.github/workflows/build-release.yml

```
@@ -63,15 +64,31 @@ jobs:
+        if: ${{ matrix.os == 'windows-latest' }}
+        uses: google-github-actions/setup-gcloud@v2
+        with:
+          version: '>=536.0.0'
+
+      - name: Authenticate to Google Cloud (Windows)
+        if: ${{ matrix.os == 'windows-latest' }}
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: ${{ secrets.GCP_SA_JSON }}
+
       - uses: ./workspaces/desktop-release-action
```

actionlint (1.7.11)

[error] invalid runner name "node12" at runs.using in "Rocket.Chat Desktop App Release Action" action defined at "/home/jailuser/git/workspaces/desktop-release-action". valid runners are "composite", "docker", "node20", and "node24". see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs

(action)